### ISSUE TITLE
About Hyperparameters

### ISSUE THREAD

**Issue by LfWhat** [YBGou/2024-ICML-TAO#3]
Is the loss function coefficient selected based on experience when testing different degradation scenarios? Is there a loss function set for other degradation scenarios?

**Comment by YBGou** [comment]
Yes, the coefficients for the loss function were selected based on experience and adjusted on some representative samples for each degradation type. In the paper, evaluations were conducted specifically on haze, noise, and low-light scenarios. For other degradation types, the loss function would need to be tuned accordingly.
